apps/vscode/vscode.talon

```diff
@@ -54,6 +54,10 @@ show shortcuts: user.vscode("workbench.action.openGlobalKeybindings")
 show shortcuts json: user.vscode("workbench.action.openGlobalKeybindingsFile")
 show snippets: user.vscode("workbench.action.openSnippets")
 
+# VSCode Snippets
+snip last: user.vscode("jumpToPrevSnippetPlaceholder")
+[snip] next: user.vscode("jumpToNextSnippetPlaceholder")
```

Comment:
This is what I use and it's great. I do wonder if it's too short to have by default? I also wonder if there should be a generic action for moving placeholders that we implement this way in VSCode cc/ @knausj85 @AndreasArvidsson

Reply:
Maybe remove the brackets?
An action is probably better as a general rule.

Reply:
I would support removing the brackets. 'Next' maybe fine for your personal setup but would conflict with mine and probably some other peoples'.

Reply:
Removed the brackets! Thanks for the feedback.